### PR TITLE
Bugfix branch - Pull request for "PowerPay-MobileApp" - Web.app

### PR DESCRIPTION
**Pull request by JendareyTechnologies**
Dear Engineering Team,

We kindly request your review of the following pull request, 
which contains code changes intended for merging into our MasterBranch.

Please review and approve these changes

Thank you for your collaboration.

Sincerely,

Senior DevOps Engineer
AK

### CODE REVIEW

**Review by akinluyigithub — APPROVED** [review]
This looks good!!

Nice Job.

Approved for merging into the MasterBranch

**Review by yemyem79 — APPROVED** [review]
It looks nice, thanks. Approved for merging into the MasterBranch.

**Review by Jimi2708 — APPROVED** [review]
This seems good to me 
Great job

Approved to be merge

**Comment by REdward22** [comment]
This looks good!

Approved for merge into the Master branch.

Ronae

File: Web.app
Comment (adeotiib) [inline Web.app:1]: Nice job

**Review by 2010jesus4ever — APPROVED** [review]

**Review by Skibanj1982 — APPROVED** [review]
Good Job!

Please merge with master branch

**Review by yemyem79 — APPROVED** [review]
Nice work, Approved.

**Comment by BrianKolawole** [comment]
Looks Good

File: Web.app
Comment (olabisigithub) [reply Web.app:1]: nice job

**Comment by 2010jesus4ever** [comment]
The code is working. Nice job guys!!!

File: Web.app
Comment (Norriday29) [reply Web.app:1]: approved

**Comment by olusayoaris** [comment]
nice one

**Review by olabisigithub — APPROVED** [review]
approved

**Review by Norriday29 — COMMENTED** [review]
good job

**Pull request merged**